requirements.txt

```diff
@@ -1,5 +1,4 @@
 signac>=0.9.0
-enum34 ; python_version < "3.4"
```

Comment:
This line is actually for Python 2.7, not 3.4 if I understand correctly. https://github.com/conda-forge/signac-flow-feedstock/blob/a99c7d3728dbae4d8534c2797d37b64fb054c509/recipe/meta.yaml#L24